packages/server/src/AccountsServer.js

```diff
@@ -507,7 +510,9 @@ export class AccountsServer {
     address = this._getFirstUserEmail(user, address); // eslint-disable-line no-param-reassign
     const token = generateRandomToken();
     await this.db.addResetPasswordToken(userId, address, token);
-    const resetPasswordUrl = `${this._options.siteUrl}/reset-password/${token}`;
+
+    const siteUrl = this._options.siteUrl || config.siteUrl;
+    const resetPasswordUrl = `${siteUrl}/reset-password/${token}`;
```

Comment:
@TimMikeladze @pradel Actually, this path this is a config from the client so I rather have that change in another PR